lib/id3c/cli/io/google.py

```diff
@@ -81,3 +81,15 @@ def extract_document_id_from_google_url(url_str: str) -> Optional[str]:
     google_docs_matches = re.match(google_docs_pattern, url.path)
 
     return google_docs_matches["document_id"] if google_docs_matches else None
+
+def get_document_details(document_id) -> dict:
```

Comment:
Could you please add a type hint for `document_id` here?

Reply:
Added.